ci/docker/install/ubuntu_clojure.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# build and install are separated so changes to build don't invalidate
+# the whole docker cache for the image
+
+set -ex
+# install libraries for mxnet's clojure package on ubuntu
+echo 'Installing Clojure...'
+
+wget https://raw.githubusercontent.com/technomancy/leiningen/stable/bin/lein
+chmod 777 lein
+sudo cp lein /usr/local/bin
+sudo apt install libcurl3
+sudo add-apt-repository ppa:timsc/opencv-3.4
```

Comment:
Is the opencv version that is in ubuntu too old? Is not clear why we are adding this to me.

Reply:
I think it's also causing some issues in CI:

http://jenkins.mxnet-ci.amazon-ml.com/blue/organizations/jenkins/incubator-mxnet/detail/PR-11205/19/pipeline

Reply:
yes - that is the issue I'm looking into. It appears that the version of the opencv that was used to build the scala jars on nexus is different than the opencv required by caffe. I'm looking into changing the unit test from the clojure package to run off of the ones generated by `make scalapkg` and `make scalainstall` instead, (which is most likely a better way to go anyway).